robottelo/helpers.py

```diff
@@ -413,3 +414,37 @@ def default_url_on_new_port(oldport, newport):
 def get_func_name(func):
     """Given a func object return standardized name to use across project"""
     return '{0}.{1}'.format(func.__module__, func.__name__)
+
+
+def get_services_status():
+    """Check if core services are running"""
+    major_version = get_host_info()[1]
+    services = (
+        'foreman-proxy',
+        'foreman-tasks',
+        'httpd',
+        'mongod',
+        'postgresql',
+        'pulp_celerybeat',
+        'pulp_resource_manager',
+        'pulp_streamer',
+        'pulp_workers',
+        'qdrouterd',
+        'qpidd',
+        'smart_proxy_dynflow_core',
+        'squid',
+        'tomcat6' if major_version == RHEL_6_MAJOR_VERSION else 'tomcat',
+    )
```

Comment:
Thank you @ldjebran, I forgot this is also being used in test_installer.py. After some checking I'd like to leave this PR as is and change test_installer to use this helper function in a separate PR.